### PR TITLE
noobify welcome demo

### PR DESCRIPTION
**Pull request by ingawei**
FOR DEMO PURPOSES ONLY, this will show a welcome tutorial everytime, but eventually this will only show for new users.
Screen Shot 2022-07-27 at 8 30 05 PM
Screen Shot 2022-07-27 at 8 33 10 PM
Screen Shot 2022-07-27 at 8 33 27 PM
Screen Shot 2022-07-27 at 8 30 12 PM

### CODE REVIEW

**Review by akrolsmir — APPROVED** [review]
These look awesome - thanks for building this out!

One more note, I'd suggest using https://tinypng.com/ to slim down the image sizes

File: web/components/onboarding/welcome.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  }
+
+  function decreasePage() {
+    if (page > 0) {
+      setPage(page - 1)
+    }
+  }
+
+  async function setUserHasSeenWelcome() {
+    // await changeUserInfo({ showWelcome: false })
```
[the remark this answers is not in the document]
Comment (ingawei) [reply web/components/onboarding/welcome.tsx:30]: done

File: web/tailwind.config.js [hunk 2]
```diff
@@ -18,6 +18,15 @@ module.exports = {
       backgroundImage: {
         'world-trading': "url('/world-trading-background.webp')",
       },
+      colors: {
+        'greyscale-1': '#FBFBFF',
```
[the remark this answers is not in the document]
Comment (ingawei) [reply web/tailwind.config.js:22]: I just think it's more consistent with the color palette Rachael designed, her grays are more purpley and I like that :) I also don't know what the  best practices are, but I just wanted to make the names distinct enough from the tailwind colors that they couldn't get confused

File: web/components/onboarding/welcome.tsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        />
+      ))}
+    </Row>
+  )
+}
+
+function Page0() {
+  return (
+    <>
+      <img
```
[the remark this answers is not in the document]
Comment (ingawei) [reply web/components/onboarding/welcome.tsx:104]: done

**Pull request merged**